conf/global.json

```diff
@@ -5,6 +5,14 @@
     "prefix": "PREFIX_GOES_HERE",
     "region": "us-east-1"
   },
+  "general": {
```

Comment:
I'm fine with `general`. The only other thing I can think of is `file_locations`